### PR TITLE
Update stac browser links

### PR DESCRIPTION
**Pull request by smohiudd**
The STAC browser links for `dev` and `staging` have been updated to the following: 

http://veda-dev-stac-browser.s3-website-us-west-2.amazonaws.com/
http://veda-staging-stac-browser.s3-website-us-west-2.amazonaws.com/

### CODE REVIEW

File: veda-apis.qmd [hunk 1]
```diff
@@ -8,15 +8,15 @@ The VEDA API source code is currently stored in a private repository: [delta-bac
 
 ### Staging (maintenance will be announced):
 
-* STAC browser (experimental): [delta-staging-stac-browser](http://delta-staging-stac-browser.s3-website-us-east-1.amazonaws.com/)
+* STAC browser (experimental): [veda-staging-stac-browser](http://veda-staging-stac-browser.s3-website-us-west-2.amazonaws.com/)
```
Comment (anayeaye) [inline veda-apis.qmd]: I think we can now drop `(experimental)` here and below because v3 was officially released

**Comment by anayeaye** [comment]
@smohiudd I _think_ that the [github workflows now render html](https://github.com/NASA-IMPACT/veda-documentation/blob/main/.github/workflows/ci.yml#L29-L32) for us now, so the only change needed in this PR is to `veda-apis.qmd` (and maybe an update to the README to reflect the new PR process). @abarciauskas-bgse is that right or should the html files still be included? Also, should we port these updates to the [new veda-docs](https://github.com/NASA-IMPACT/veda-docs/blob/main/veda-apis.qmd)?

**Comment by abarciauskas-bgse** [comment]
@anayeaye @smohiudd yes please make these changes to https://github.com/NASA-IMPACT/veda-docs/blob/main/veda-apis.qmd

**Comment by anayeaye** [comment]
@abarciauskas-bgse do we just make changes to the qml file or do the rendered html updates also need to be committed in the PR? Or does it matter?

**Comment by abarciauskas-bgse** [comment]
@anayeaye if we're just changing markdown cells or markdown files, I believe all those changes should be rendered to html via the github actions: https://github.com/NASA-IMPACT/veda-docs/blob/main/.github/workflows/quarto-publish.yml#L25. If changes are made to a jupyter notebook code cell, that code is _not_ re-executed via github actions before being rendered to html. You can do this with the `--execute` option and I would love to execute the code blocks via github actions, but some of the code is prohibitively time-consuming (and also some of the notebooks must be run with access to the data + dask)

**Comment by smohiudd** [comment]
> @smohiudd I _think_ that the [github workflows now render html](https://github.com/NASA-IMPACT/veda-documentation/blob/main/.github/workflows/ci.yml#L29-L32) for us now, so the only change needed in this PR is to `veda-apis.qmd` (and maybe an update to the README to reflect the new PR process). @abarciauskas-bgse is that right or should the html files still be included? Also, should we port these updates to the [new veda-docs](https://github.com/NASA-IMPACT/veda-docs/blob/main/veda-apis.qmd)?

Thanks @anayeaye, I'll redo the pull request without the rendered html files. Will make the same changes in `veda-docs` as well.

**Review by anayeaye — APPROVED** [review]

**Pull request merged**